scripts/start-application.mjs

```diff
@@ -105,16 +105,16 @@ inquirer
         const appId = await getApplicationId(packageName);
         await concurrently(
           [
-            `VITE_CONTAINER_APP=${appId} yarn workspace ${containerPackageName} run start:dev`,
-            `CONTAINER=1 yarn workspace ${packageName} run start:dev`,
+            `VITE_CONTAINER_APP=${appId} turbo run start --filter=${containerPackageName}`,
+            `CONTAINER=1 turbo run start --filter=${packageName}`,
```

Comment:
On my computer I have a lot of issues when using turbo due to a lot of files access, I think we need to do some performances checks before migrating

Reply:
What kind of issues & on which platform ?

If it's too many disk IO, @kqesar i would suggest disabling turbo daemon as i find the performance gain to be negligible and i did encouter some issues on OSX with it.

Regarding benchmarks, we did some test on our dev environement, starting an application the fist time (without cache) is taking the same time as before (~30s-1min). However starting an application a second time once the turbo cache is available, takes only a few seconds (<5s)

Reply:
on Linux my CPU was at 100% for 10 minutes only due to the daemon

Reply:
I will do a test this afternoon on this branch to check, it might be due to an old turbo versions that might have been fixed

Reply:
After some tests this issues doesn't exist anymore, everything is fine except some errors on turbo daemon, but it is not specific to this project